src/app/zap-templates/zcl/data-model/silabs/types.xml

```diff
@@ -209,14 +209,14 @@ limitations under the License.
   </enum>
   <enum name="HueDirection" type="ENUM8">
     <cluster code="0x0300"/>
-    <item name="ShortestDistance" value="0x0"/>
-    <item name="LongestDistance" value="0x1"/>
+    <item name="Shortest distance" value="0x0"/>
+    <item name="Longest distance" value="0x1"/>
```

Comment:
Disagree with adding spaces, at any point in the game... This really causes more enforcement of space conversion rules to be used by any tool attempting to make use of the XMLs.

Of course, right now, we can't expect others than ZAP to use this particular XML, but it does beg the question overall of why would we encourage the spec to keep these bad identifiers that also, in a large sense, make it harder to uniformly "scrape" the spec and implement naming in software.

Reply:
OK, I undid these changes and will try to fix this in the spec instead.